apps/dashboard/app/javascript/packs/batch_connect.js

```diff
@@ -191,46 +191,48 @@ function addHideHandler(optionId, option, key,  configValue) {
  *        data-max-num-cores-for-cluster-annie-oakley: 42
  *      ]
  */
-function addMinMaxForHandler(optionId, option, key,  configValue) {
-  optionId = String(optionId || '');
+function addMinMaxForHandler(subjectId, option, key,  configValue) {
+  subjectId = String(subjectId || '');
```

Comment:
Lot of renames here. We need to refactor this a bit to follow the `subject & object` pattern like the Cnglish language sentence structure.

That is if `node_type changes num_cores`, `node_type` is the subject and `num_cores` is the object.